File: VB/Q253018/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraBars

Namespace Q253018
	Partial Public Class Form1
		Inherits Form

		Private helper As BarHelper

		Public Sub New()
			InitializeComponent()
			helper = New BarHelper(barManager1)
			helper.ExpandMenuOnMouseHover = True
		End Sub

		Protected Overrides Sub OnFormClosing(ByVal e As FormClosingEventArgs)
			MyBase.OnFormClosing(e)
			helper.ExpandMenuOnMouseHover = False
		End Sub
	End Class
End Namespace

File: CS/Q253018/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraBars;

namespace Q253018 {
    public partial class Form1 :Form {

        BarHelper helper;

        public Form1() {
            InitializeComponent();
            helper = new BarHelper(barManager1);
            helper.ExpandMenuOnMouseHover = true;
        }

        protected override void OnFormClosing(FormClosingEventArgs e)
        {
            base.OnFormClosing(e);
            helper.ExpandMenuOnMouseHover = false;
        }
    }
}